### DISCUSSION TITLE
Reports Module

### DISCUSSION THREAD

**Discussion by ad1rie1** [YetiForceCompany/YetiForceCRM#14033] [Wishlist]
Hello, 

Removed a long time ago ;) A good report module is helpful for management ;)

see: https://github.com/YetiForceCompany/YetiForceCRM/issues/6538

Dear;

**Comment by simple-AMS** [comment]
@ad1rie1 According to the powers that be, reporting is achieved via widgets. According to every business owner on the planet, they require the actual ability to run and generate various report types. Good luck!

**Reply by ad1rie1** [reply]
yes i have added this feature request as backlog :p 
i use Metabase for now and it does the job until maybe feature appear in yeti ;)

**Reply by simple-AMS** [reply]
> yes i have added this feature request as backlog :p
> i use Metabase for now and it does the job until maybe feature appear in yeti ;)

You have managed to integrate the data from Yeti into Metabase? If so, care to outline how you were able to achieve this please?

**Reply by ad1rie1** [reply]
Install Metabase, connect directly to your crm sql, and make data you want to, it's very user friendly

**Reply by simple-AMS** [reply]
Thanks @ad1rie1
